pkg/cmd/server/api/types.go

```diff
@@ -14,12 +16,17 @@ const (
 )
 
 var (
-	KnownKubernetesAPILevels   = []string{"v1beta1", "v1beta2", "v1beta3", "v1"}
-	KnownOpenShiftAPILevels    = []string{"v1beta1", "v1beta3", "v1"}
-	DefaultKubernetesAPILevels = []string{"v1beta3", "v1"}
-	DefaultOpenShiftAPILevels  = []string{"v1beta3", "v1"}
-	DeadKubernetesAPILevels    = []string{"v1beta1", "v1beta2"}
-	DeadOpenShiftAPILevels     = []string{"v1beta1"}
+	KnownKubernetesAPILevels            = []string{"v1beta1", "v1beta2", "v1beta3", "v1"}
+	KnownOpenShiftAPILevels             = []string{"v1beta1", "v1beta3", "v1"}
+	KnownKubernetesStorageVersionLevels = []string{"v1"}
+	KnownOpenShiftStorageVersionLevels  = latest.Versions
+	DefaultKubernetesAPILevels          = []string{"v1"}
+	DefaultOpenShiftAPILevels           = []string{"v1"}
+	DefaultOpenShiftStorageVersionLevel = latest.Versions[0]
+	DeadKubernetesAPILevels             = []string{"v1beta1", "v1beta2", "v1beta3"}
+	DeadOpenShiftAPILevels              = []string{"v1beta1", "v1beta3"}
+	DeadKubernetesStorageVersionLevels  = []string{"v1beta3"}
+	DeadOpenShiftStorageVersionLevels   = []string{"v1beta1", "v1beta3"}
```

Comment:
leave the API lists in a group and put the storage lists in their own group
doc what Known... and Dead... means and are used for